openedx/core/lib/tests/test_xblock_utils.py

```diff
@@ -173,28 +173,6 @@ def test_replace_static_urls(self, course_id, anchor_tag):
         self.assertIsInstance(test_replace, Fragment)
         self.assertEqual(test_replace.content, anchor_tag)
 
-    @ddt.data('course_mongo', 'course_split')
-    def test_grade_histogram(self, course_id):
```

Comment:
Quoting @nasthagiri 
> The issue with the `block_cache` was because `block_cache` is defined in `CACHES` in `lms/envs/test.py`, but not in `cms/envs/test.py`.